classes/PodsUI.php

```diff
@@ -3660,7 +3789,20 @@ public function filters_popup() {
 								</label>
 
 								<span class="pods-ui-posts-filter<?php esc_attr_e( strlen( $value ) < 1 ? ' pods-hidden' : '' ); ?>">
-								<?php echo PodsForm::field( 'filter_' . $filter, $value, 'pick', $options ); ?>
+								<?php
+								// Prevent p div issues.
+								echo str_replace(
```

Comment:
:no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'str_replace'.

<sub>hash: d0c07ff95cf9f5dd46bcaeca7f351633</sub>